testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move

```diff
@@ -0,0 +1,85 @@
+// Copyright Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+module 0xABCD::smart_table_picture {
```

Comment:
someone who knows move needs to review this

Reply:
I assume we copied from certain bad contract?

Reply:
yup 

https://github.com/aptos-labs/graffio/blob/main/move/sources/canvas/token.move#L223